### PR TITLE
Configure domains per traefik labels

### PR DESCRIPTION
**Pull request by rpsu**
Correnct faulty nginx router rules fro Traefik (only one Host() rule works), and gather all configured router hostnames even if set in the same line.

### CODE REVIEW

File: docker/docker-compose.skeleton.yml [hunk 1]
```diff
@@ -42,8 +42,7 @@ services:
     build: ./docker/build/nginx
     container_name: "${PROJECT_NAME}_nginx"
     labels:
-      - "traefik.http.routers.${PROJECT_NAME}--web-ui.rule=Host(`${LOCAL_DOMAIN}`)"
-      - "traefik.http.routers.${PROJECT_NAME}--web-ui.rule=Host(`www.${LOCAL_DOMAIN}`)"
+        - "traefik.http.routers.${PROJECT_NAME}--web-ui.rule=Host(`${LOCAL_DOMAIN}`, `www.${LOCAL_DOMAIN}`)"
```
Comment (kekkis) [inline docker/docker-compose.skeleton.yml]: This indentation is 3 spaces whereas in the other docker-compose files it's only 2.
```suggestion
       - "traefik.http.routers.${PROJECT_NAME}--web-ui.rule=Host(`${LOCAL_DOMAIN}`, `www.${LOCAL_DOMAIN}`)"
```

File: docker/scripts/ld.command.configure-network.sh [hunk 2]
```diff
@@ -27,7 +27,7 @@ function ld_command_configure-network_exec() {
     DOMAIN_SET=$(egrep -e $LOCAL_IP'\s*([a-z0-9\.-]*\s?\s*)*('$LOCAL_DOMAIN')' /etc/hosts)
     if [ "$LOCAL_DOMAIN" != "localhost" ]; then
         if [ -z "$DOMAIN_SET" ]; then
-            SUBDOMAINS=$(grep '=Host' docker-compose.yml | cut -d'`' -f2 | cut -d'.' -f1 | grep -v '\$' | xargs -I % echo %.${LOCAL_DOMAIN} | xargs)
+            SUBDOMAINS=$(egrep 'traefik\.[a-z]*\.routers..*\.rule\=Host\(' docker-compose.yml | grep -o '[a-z0-9\.]*\${LOCAL_DOMAIN\}'  | cut -d'.' -f1 | grep -v '^\$' | xargs -I % echo %.${LOCAL_DOMAIN} | xargs)
```
Comment (kekkis) [inline docker/scripts/ld.command.configure-network.sh]: That is one big pipe. Can we get a breakdown or at least a comment on what is going on there, for those of us that are lazy pipe readers? That would improve the maintainability IMO.

Comment (rpsu) [reply docker/scripts/ld.command.configure-network.sh]: absolutely

**Comment by rpsu** [comment]
@kekkis is this close to what you were thinking?
https://github.com/Exove/local-docker/pull/154/commits/a6a9ec4484bcfcdcf86163382f8b3f0c34c3c0db

**Comment by kekkis** [comment]
> @kekkis is this close to what you were thinking?

Yes, thanks!

**Pull request merged**